babyshop_app/Dockerfile

```
@@ -0,0 +1,33 @@
+# syntax=docker/dockerfile:1
+FROM python:3.12-slim
+
+# Set environment variables to optimize Python
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1
+
+# Set work directory
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends gcc libpq-dev python3-distutils \
```

```suggestion
    && apt-get install -y --no-install-recommends gcc python3-distutils \
```

You are using a sqlite structure in your project, so you dont postgres dependencies. gcc is maybe necessary for compiling packages in your requirements.txt